Documentation/Contributors/BuildGuide/README.md

```diff
@@ -63,19 +62,8 @@ Cesium uses [npm modules](https://docs.npmjs.com/getting-started/what-is-npm) fo
 npm install
 ```
 
-Once all modules have been installed, run `npm run build` to actually build the code:
-
-```bash
-npm run build
-```
-
-Alternatively, if you want to edit source files and see the changes reflected for testing, use `npm run build-watch`.
-
-```bash
-npm run build-watch
-```
-
-Cesium ships with a simple HTTP server for testing, run `npm start` after building to use it:
+Cesium ships with a simple HTTP server for testing.
+Once all modules have been installed, run `npm start` to use it:
```

Comment:
This does not seem to be working for modifications of existing specs. With `build-watch`, I can modify a file and re-run the spec with the updated version.

Reply:
@ggetz I'm not sure if the latest fix was correct because now, upon initially running `npm install; npm start`, the specs are not built.

Reply:
Apologies. There was an issue where the specs were getting built twice in quick succession unnecessarily. I fixed the issue and added a comment.